tests/SprykerEcoTest/Yves/Payolution/_support/_generated/PayolutionYvesTesterActions.php

```diff
@@ -1,17 +1,10 @@
-<?php  //[STAMP] f58a5acd49d6fbf54fd51ffe98c733b1
+<?php  //[STAMP] 253cfabd0d746f7f4411f5dd8cac5d9d
```

Comment:
I guess you need to add this class in gitignore and remove it from repository as it's generated anyway.